decoder_test.go

```diff
@@ -2214,9 +2215,75 @@ func TestRequiredFieldsCannotHaveDefaults(t *testing.T) {
 
 }
 
+func TestInvalidDefaultElementInSliceRaiseError(t *testing.T) {
+	type D struct {
+		A []int      `schema:"a,default:0|notInt"`
+		B []bool     `schema:"b,default:true|notInt"`
+		C []*float32 `schema:"c,default:1.1|notInt"`
```

Comment:
I missed on this in the first pass: slices of pointers are not supported (I will try to add a custom error message in a different PR). As you can see, it is mentioned here in the readme: https://github.com/gorilla/schema/blob/main/README.md?plain=1#L112  I think all the slices of pointers are not necessary, this would avoid doing this check in the loop:
```
if ferr.Error() == pErrMsg {
				continue
			}
```

Reply:
thanks for review!

Reply:
welcome.